Statut.md

```diff
@@ -0,0 +1,169 @@
+
+§17 Odpowiedzialność za zobowiązania
+Fundacja odpowiada za swoje zobowiązania całym majątkiem.
+
+Rozdział IV: Organy Fundacji
+
+§18 Zarząd
+1. Organem Fundacji jest Zarząd Fundacji.
+2. Zarząd kieruje działaniami Fundacji i reprezentuje ją na zewnątrz.
+
+§19 Fundatorzy
+1. Fundator do wykonywania swoich uprawnień może powołać pełnomocnika.
+2. W przypadku śmierci Fundatora uprawnienia przewidziane w niniejszym statucie dla Fundatora, wykonuje jego pełnoletnie dziecko, łącznie pełnoletnie dzieci lub do czasu uzyskania pełnoletności ich prawni opiekunowie. Jeżeli Fundator nie posiadał dzieci, jego obowiązki wykonuje żyjący małżonek, albo łącznie rodzeństwo Fundatora, albo łącznie pełnoletnie dzieci rodzeństwa Fundatora lub do czasu uzyskania pełnoletności ich prawni opiekunowie.
+
+§20 Członkowie zarządu #DoDyskusji
+1. Członkowie Zarządu Fundacji powoływani i odwoływani są przez Fundatorów w drodze zgodnego oświadczenia na czas wskazany w akcie powołania.
+2. Fundator może zostać członkiem Zarządu Fundacji.
+3. Zarząd Fundacji liczy od ... do ... (???) członków.
+4. Członkowie Zarządu Fundacji mogą otrzymywać wynagrodzenie z tytułu pełnienia swoich funkcji.
+5. Członkostwo w Zarządzie Fundacji ustaje w przypadku odwołania, pisemnej rezygnacji z członkostwa lub śmierci Członka.
+
+§21 Zadania zarządu
+1. Zarząd Fundacji kieruje działalnością Fundacji i reprezentuje ją na zewnątrz.
+2. Do zadań Zarządu należy w szczególności:
+	1) Kierowanie bieżącą działalnością Fundacji;
+	2) Podejmowanie decyzji w sprawie prowadzenia działalności gospodarczej;
+	3) realizacja celów statutowych Fundacji,
+	4) uchwalanie rocznych i wieloletnich planów działania Fundacji,
+	5) przygotowywanie i uchwalanie sprawozdań z działalności Fundacji,
+	6) zarządzanie majątkiem Fundacji.
+3. Zarząd Fundacji podejmuje decyzje na posiedzeniach w formie uchwał zwykłą większością głosów.
+4. O posiedzeniu muszą być powiadomieni wszyscy członkowie Zarządu z wyprzedzeniem co najmniej 7 dni kalendarzowych.
+5. Zarząd może powoływać pełnomocników do kierowania wyodrębnioną sferą spraw należących do zadań Zarządu Fundacji.  
+
+§22 Prezes i wiceprezes
+1. Zarządowi przewodniczy Prezes Zarządu Fundacji, wskazywany przez Fundatorów w akcie powołania Zarządu.
+2. Prezes Zarządu Fundacji zwołuje posiedzenia Zarządu Fundacji z własnej inicjatywy, bądź na wniosek każdego z pozostałych członków Zarządu Fundacji, nie rzadziej niż raz na pół roku.
+3. Prezes Zarządu Fundacji może powołać Wiceprezesa Zarządu Fundacji spośród pozostałych członków Zarządu, który wykonuje obowiązki Prezesa podczas jego niemożności działania lub w zakresie mu powierzonym.
+
+§23 Składanie oświadczeń woli i zwykły zarząd
+1. Do składania oświadczeń woli w imieniu Fundacji w sprawach majątkowych i niemajątkowych w zakresie zwykłego zarządu uprawniony jest Prezes Zarządu Fundacji samodzielnie.
```

Comment:
Samodzielnie?

Reply:
Przez zwykły zarząd rozumie się na ogół takie codzienne czynności, gdzie konieczność głosowania co chwilę nad każdą sprawą paraliżowałaby normalne funkcjonowanie. Władzę prezesa można najlepiej zwiększyć lub zmniejszyć przez definicje spraw przekraczających zwykły zarząd (punkt 3 poniżej, linijki 139-144). To co przekracza zwykły zarząd wymaga głosowania, więc jeżeli w/w definicje dają prezesowi duże kompetencje, to jest bardziej autokracja, jeżeli dają mniejsze "samodzielne" kompetencje to więcej decyzji siłą rzeczy musi zapaść kolektywnie przez głosowanie wśród członków zarządu. Wiadomo, że ta struktura ma charakter formalny i w praktyce i tak sporo decyzji będzie padać wspólnie szczególnie kiedy ekipa jest nieduża, a wszyscy są w miarę zgodni. Ale zgodnie z planem ekipa nie zawsze będzie mała i nie zawsze będzie zgoda, a wtedy to właśnie takie zapisy formalne tłumaczą jak powinno być.

Reply:
Czy możemy to zliberalizować do Człona Zarządu? Co o tym myślicie? (piszę to jeszce przed przeczytaniem punktu 3).

Reply:
1. Nie wydaje mi się, że można. Fundacja ma prezesa po to, żeby miał kompetencje zwykłego zarządu. Nie jestem pewien, czy to się da zredefiniować. Trzeba wrócić do tej dyskusji BARDZO OTWARTE